### DISCUSSION TITLE
Buscando equipo

### DISCUSSION THREAD

**Discussion by sjarmijos** [Launch-X-Latam/MisionFrontEnd#517] [Team Posts]
Hola mi nombre es Santiago yo voy empezando con el mundo del desarrollo tengo conocimiento básico de HTML, CSS lo que más he manejado en este mundo es Bootstrap y un poco de Python con Django, muy dispuesto a seguir aprendiendo tanto de habilidades de programación como también de trabajo en equipo, si alguien gusta incluirme en su grupo se los agradecería

**Comment by DavisonH** [comment]
faltan tres persona que deseen unirse, este es el correo : lx12963@innovaccion.mx

**Comment by snowingt** [comment]
> faltan tres persona que deseen unirse, este es el correo : lx12963@innovaccion.mx

Hey me gustaria participar

**Comment by 4Ndrs91** [comment]
> faltan tres persona que deseen unirse, este es el correo : lx12963@innovaccion.mx

Me gustaría unirme este es mi correo lx16112@innovaccion.mx

**Comment by Josephr1318** [comment]
Me gustaría unirme si aún hay cupo mi correo es; ***@***.***—

**Comment by DavisonH** [comment]
@snowingt @Josephr1318  bro si te quieres unir pasa tu correo de teams

**Comment by sjarmijos** [comment]
> @snowingt @Josephr1318 bro si te quieres unir pasa tu correo de teams

Si aun hay espacio este es mi correo lx08946@innovaccion.mx

**Comment by cardenas14** [comment]
me podria unir ahi dejo mi correo 
lx05913@[innovaccion.mx](http://innovaccion.mx/)
